core/src/main/scala/cats/syntax/parallel.scala

```diff
@@ -237,35 +237,69 @@ final class ParallelUnorderedFlatSequenceOps[T[_], M[_], A](private val tmta: T[
     Parallel.parUnorderedFlatSequence(tmta)
 }
 
-final class ParallelApOps[M[_], A](private val ma: M[A]) extends AnyVal {
+final class ParallelApOps[M[_], A](protected val ma: M[A]) extends AnyVal with ParallelApOpsBinCompat0[M, A] {
 
-  def &>[B](mb: M[B])(implicit P: Parallel[M]): M[B] =
+  @deprecated("use a NonEmptyParallel-constrained version instead", "2.7.1")
```

Comment:
The project version is set to "2.8.0-SNAPSHOT", if I'm not mistaken. I guess it'd be better so set versions in these `@deprecated` to "2.8.0"